lib/project_types/extension/features/argo_serve.rb

```diff
@@ -22,6 +23,11 @@ def call
         end
       end
 
+      def resource_url
+        project = ExtensionProject.current(force_reload: true)
+        super || project.resource_url
```

Comment:
Nit: Since we're using `force_reload: true`, I would implement this method as the one-liner:

```ruby
super || ExtensionProject.current(force_reload: true).resource_url
```

This way, we only reload the current project if it's actually needed (i.e. if `super` is a falsely value).

Reply:
Cool, I didn't know you could do that! Done!